FlowCal/excel_ui.py

```diff
@@ -562,6 +562,53 @@ def process_samples_table(samples_table,
 
     return samples
 
+def add_beads_stats(beads_table, beads_samples):
```

Comment:
I like that this function name is verbose and consistent with other function names (e.g. `process_beads_table`). Might consider updating `add_stats` to `add_sample_stats`.